docs/evaluation_plan.md

```diff
@@ -0,0 +1,128 @@
+The $WIS^{aggregate}_{h,d,l,m}$ term should roughly account for the average forecast difficulty of a particular model at that particular forecast location, forecast date, and horizon.
+
+We can roughly interpret $\beta$ as the average effect of local forecasting on forecast performance compared to the aggregate forecast, and the $s(model, bs = "re")$ term as the average additional effect across all locations, forecast dates, and horizons from a particular model.
+
+The $s(location, bs = "re")$ term tells us about the additional effect of location on the relative value of local forecasting, whereas the $s(forecastdate, by = location)$ term tells us about the effect of the forecast date and location combinations on the relative value of local forecasting.
+
+We will plot the partial effects of each of these components in order to understand the drivers of differences in forecast performance between local and aggregate forecasts.
+
+### Retrospective analysis: local model comparison
+This analysis will compare performance between the different retrospectively produced local forecasts.
+
+We will use the scores from the previous section, focusing only on the local scores.
+Relative WIS will be computed relative to the baseline local model.
+
+Again, we will examine the relative and absolute WIS scores across multiple stratifications: overall, by nowcast horizon, and by location.
+
+#### Model-based evaluation for comparing across local models
+Again, and in a more similar vein to Sherratt et al, we will perform a model-based evaluation to account for confounding variables impacting the forecast performance of each model
+
+We will include the following confounding variables:
+- location
+- forecast date and location
+- horizon
+- epidemic phase
+- model
+
+Resulting in the following model formulation:
+
+```math
+WIS^{local}_{h,d,l,m} \sim \beta + s(location, bc = "re") + s(forecast_date, by = "location") + s(horizon, k) + s(epidemic_phase, bs = "re") + s(model, bs = "re")
+```
+Where $h$ is the forecast horizon (from -1 to 4 weeks), $d$ is the forecast date, $l$ is the location of the forecast (the borough or metro area), and $m$ is the model.
+The goal of this analysis will be to estimate the effect of the model (via the $s(model, bs = "re")$ term) while taking into account the many additional confounding variables that contribute to forecast performance.
+
+### Real-time analysis: local model comparison
+Using the quantiles already submitted to the Hub from the models in real-time we will score forecasts using the WIS.
+Relative WIS will be computed relative to the baseline model.
+
+Because not all models were submitted for all forecast dates and locations, with only a handful of models submitted for the complete set of forecast dates, we will not compute overall and by horizon, location, and forecast date averages because of potential bias in these summary metrics due to different forecasts being present within each model's set of forecasts.
+
```

Comment:
There's a workaround for this and it is using geometric average pair relative comparison:  https://epiforecasts.io/scoringutils/reference/add_relative_skill.html